ui/app/components/streaming-file.js

```diff
@@ -41,8 +50,42 @@ export default class StreamingFile extends Component.extend(WindowResizable) {
     }
   }
 
+  scrollHandler() {
+    const cli = this.element;
+    window.requestAnimationFrame(() => {
```

Comment:
Should this be wrapped in an `if` to check on the value of `requestFrame`? It looks like the value of that field is updated but never checked.

Reply:
Haaaaaaaa yep. Thank you for catching this.